LICENSE

```diff
@@ -1,3 +1,13 @@
+Copyright (c) 2021-present Firezone, Inc.
+
+Portions of this software are licensed as follows:
+
+* All content residing under the "elixir/" directory of this repository, if that directory exists, is licensed under the "Elastic License 2.0" license defined in "elixir/LICENSE".
+* All third party components incorporated into the Firezone Software are licensed under the original license provided by the owner of the applicable component.
+* Content outside of the above mentioned directories or restrictions above is available under the "Apache 2.0 License" license as defined below.
```

Comment:
Alternatively, we could move/copy the Apache License to all sub directories of the various components, i.e. one for `rust/`, `kotlin/`, `swift/` etc.

That is "closer" to how things would be if we wouldn't have a mono-repo.

Reply:
One advantage would be that you don't have to modify license texts.

Reply:
Updated to add clarifications to the README so GitHub's license finder works. I intentionally kept it simple for now with only two license files since that should cover us.